helpers/middleman/space.rb

```diff
@@ -1,38 +1,36 @@
-load File::expand_path('./page.rb', __dir__)
+load File.expand_path('./page.rb', __dir__)
```

Comment:
Why?

Reply:
maybe it auto fixed?

Reply:
will change it back

Reply:
If it auto-fixes, keep it, no worries if it actually works.  Not confident enough about Ruby to tell you the difference, but wouldn't be surprised if both worked.

Reply:
might just change it back in case it does break something